CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,43 @@
+# Code of Conduct
```

Comment:
Next time it is better to do all these changes on the devel branch and then merge it to master.
It helps to keep branches clean and nice